CefSharp/CefSharpSettings.cs

```diff
@@ -41,5 +41,10 @@ static CefSharpSettings()
         /// the event handlers are hooked in the static constructor for the ChromiumWebBrowser class
         /// </summary>
         public static bool ShutdownOnExit { get; set; }
+
+        /// <summary>
+        /// The proxy options that will be used for all connections
```

Comment:
Also say that `GetAuthCredentials` won't be called, for a proxy server that matches the `IP`